### PR TITLE
adding efmtool_link, optlang_enumerator

### PR DESCRIPTION
**Pull request by axelvonkamp**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by axelvonkamp** [comment]
I am willing to be listed as maintainer.

**Comment by axelvonkamp** [comment]
@conda-forge/help-python, ready for review!
I have run  ```conda smithy recipe-lint --conda-forge .``` on both recipes and get " . is in fine form". I cann0t figure out why the conda-forge-linter here gives an error.

File: recipes/efmtool_link/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python >=3.7,<3.11
+    - numpy
+    - jpype1
+    - cobra >=0.26.3
+    - psutil
+
+test:
+  imports:
+    - efmtool_link
```
Comment (xhochy) [inline recipes/efmtool_link/meta.yaml:31]: Please add a pip check call here

File: recipes/optlang_enumerator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - numpy
+    - scipy
+    - optlang
+    - cobra >=0.26.3
+    - sympy >=1.12
+    - efmtool_link >=0.0.7
+
+test:
+  imports:
+    - optlang_enumerator
```
Comment (xhochy) [inline recipes/optlang_enumerator/meta.yaml:32]: Pip check

**Review by xhochy — CHANGES_REQUESTED** [review]
To fix the linter, merge main.

File: recipes/efmtool_link/meta.yaml (hunk 1 above)
Comment (axelvonkamp) [reply recipes/efmtool_link/meta.yaml:36]: Done.

File: recipes/optlang_enumerator/meta.yaml (hunk 2 above)
Comment (axelvonkamp) [reply recipes/optlang_enumerator/meta.yaml:32]: Done

**Pull request merged**